### PR TITLE
Renamed several files in the covidcast signals folder

### PR DESCRIPTION
**Pull request by tinatownes**
Removed -inactive, and renamed quidel files to distinguish covid and flu.

closes|addresses 

### Prerequisites:

- [X] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [X] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [ ] Build is successful
- [ ] Code is cleaned up and formatted

### CODE REVIEW

**Review by nmdefries — COMMENTED** [review]
The actual changes look good, but I realized that these pages may be linked elsewhere, [e.g.](https://github.com/search?q=org%3Acmu-delphi%20delphi-epidata%2Fapi%2Fcovidcast-signals%2Fquidel.html&type=code). We'd need to update those URLs too to make sure we don't break the links.

Based on the example search results above, this will span multiple repositories, so may be easier for me to do. I think it will affect quidel more than the other pages because the original renaming from, e.g., "safegraph" -> "safegraph-inactive" probably wasn't accompanied by link renaming in other repos.

**Comment by nmdefries** [comment]
Not breaking links to doc pages was easiest by keeping the active (covid) quidel doc file named `quidel.md` instead of `quidel-covid.md`; the only other link that needed to be updated was an `.../indicator-combination-inactive.html` in this repo. No other outdated URLs exist in this repo or other Delphi repos.

**Review by nmdefries — APPROVED** [review]

File: docs/api/covidcast-signals/quidel.md [hunk 1]
```diff
@@ -159,5 +159,5 @@ not enough samples can be filled in from the parent state for smoothed signals s
 no data is reported for that area on that day; an API query for all reported geographic areas on
 that day will not include it.
 
-## Flu Tests (inactive)
-These signals were updated until May 19, 2020. Documentation is still available on the [inactive Quidel page](quidel-inactive.md).
\ No newline at end of file
+## Flu Tests
+These signals are inactive. They were updated until May 19, 2020. Documentation is still available on the [inactive Quidel page](quidel-inactive.md).
```
Comment (melange396) [inline docs/api/covidcast-signals/quidel.md:163]: this one was just renamed, right?
```suggestion
These signals are inactive. They were updated until May 19, 2020. Documentation is still available on the [Quidel flu page](quidel-flu.md).
```

Comment (nmdefries) [reply docs/api/covidcast-signals/quidel.md:163]: Thanks for pointing that out! Fixed in https://github.com/cmu-delphi/delphi-epidata/pull/1480

File: src/server/endpoints/covidcast_utils/db_sources.csv [hunk 2]
```diff
@@ -19,7 +19,7 @@ safegraph,safegraph-weekly,SafeGraph (Weekly),[SafeGraph](https://docs.safegraph
...
 google-survey,google-survey,Google Symptom Surveys,"Delphi ran symptom surveys using a Google tool which collects responses through publisher websites, Google's Opinions Reward app, and similar applications. No longer updated after May 15, 2020.",smoothed_cli,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/google-survey.html)
-indicator-combination,indicator-combination-nmf,Statistical Combination (NMF),"This source provides signals which are statistical combinations of other sources, calculated by Delphi. It is not a primary data source. No longer updated after March 17, 2021.",nmf_day_doc_fbs_ght,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/indicator-combination-inactive.html)
+indicator-combination,indicator-combination-nmf,Statistical Combination (NMF),"This source provides signals which are statistical combinations of other sources, calculated by Delphi. It is not a primary data source. No longer updated after March 17, 2021.",nmf_day_doc_fbs_ght,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/indicator-combination.html)
```
Comment (melange396) [inline src/server/endpoints/covidcast_utils/db_sources.csv:22]: @tinatownes did you put these changes into the gsheet too?

Comment (nmdefries) [reply src/server/endpoints/covidcast_utils/db_sources.csv:22]: Thanks for the reminder! @tinatownes, I'll leave this to you. This needs to be changed on the "Data sources" tab.

**Pull request merged**